lib/config/definitions.ts

```diff
@@ -412,7 +412,7 @@ const options: RenovateOptions[] = [
       'Any text added here will be placed first in the Dependency Dashboard issue body.',
     type: 'string',
     default:
-      'This issue contains a list of Renovate updates and their statuses.',
+      'This issue contains a list of Renovate updates and their statuses. Click one checkbox at a time, as Renovate will skip any you clicked too slowly.',
```

Comment:
🤔

This is only true when Renovate is configured to listen on hooks like issue updated.

For self-hosted Renovate which is only running on schedule this can only happen when Renovate is running on the repo while clicking the checkboxes.

Reply:
I don't know which option is better:

1. **Show warning** which is wrong in some cases. like self-hosted users with a schedule, but works for the hosted users
2. **Don't show warning** and end up with confused users: "I clicked 200 things, why do I only get 1 update PR... 😕"